docs/blog/pydantic-v2.md

```diff
@@ -0,0 +1,221 @@
+
+The plan is to have pydantic V2 released within 3 months of full time work 
+(again, that'll be sooner if I can continue to work on it full time).
+
+Before pydantic V2 can be released, we need to released pydantic v1.10 - there are lots of changes in the main
+branch of pydantic contributed by the community, it's only fair to provide a release including those changes,
+many of them will remain unchanged for V2, the rest will act as a requirement to make sure pydantic V2 includes
+the capabilities they implemented.
+
+The basic road map for me is as follows:
+1. implement a few more critical features in pydantic-core
+2. release V0.1 of pydantic-core
+3. work on getting pydantic V1.10 out - basically merge all open PRs that are finished
+4. release pydantic V1.10
+5. delete all stale PRs which didn't make it into V1.10, apologise profusely to their authors who put their valuable
+  time into pydantic only to have their PRs closed :pray:
+6. change the main branch of pydantic to target v2
+7. start tearing pydantic code apart and see how many existing tests can be made to pass
+8. rinse, repeat
+9. release pydantic V2 :tada:
+
+# Introduction
+
+Pydantic began life as an experiment in some code in a long dead project.
+I ended up making the code into a package and releasing it.
+It got a bit of attention on hacker news when it was first released, but started to get really popular when
+Sebastian Ramirez used it in FastAPI.
+Since then the package and its usage have grown enormously.
+The core logic however has remained relatively unchanged since the initial experiment.
+It's old, it smells, it needs to be rebuilt.
+The release of version 2 is an opportunity to rebuild pydantic and correct many things that don't make sense.
+
+Much of the work on V2 is already done, but there's still a lot to do.
+Now seems a good opportunity to explain what V2 is going to look like and get feedback from uses.
+
+## Headlines
+
+For good and bad, here are some of the biggest changes expected in V2.
+
+The core validation logic of pydantic V2 will be performed by a separate package 
+[pydantic-core](https://github.com/samuelcolvin/pydantic-core) which I've been building over the last few months.
+
+*pydantic-core* is written in Rust using the excellent [pyo3](https://pyo3.rs/) library which provides rust bindings
+for python.
+
+**Note:** the python interface to pydantic shouldn't change as a result of using pydantic-core, instead
+pydantic will use type annotations to build a schema for pydantic-core to use.
+
+pydantic-core is usable now, albeit with a fairly unintuitive API, if you're interested, please give it a try.
+
+pydantic-core provides validators for all common data types, 
+[see a list here](https://github.com/samuelcolvin/pydantic-core/blob/main/pydantic_core/_types.py#L291).
+Other, less commonly used data types will be supported via validator functions.
+
+### Performance :smile:
+
+As a result of the move to rust for the validation logic 
```

Comment:
Did. you move pydantic-core to rust because of performance reasons? Up to here you haven't shared the reasoning for that change. It might be helpful to say "We desire 10x performance, candidates were A, B.  I chose rust because performance, Y, Z."

Reply:
done.